bokeh/settings.py

```diff
@@ -43,12 +44,19 @@ def minified(self, default=None):
     def log_level(self, default=None):
         return self._get_str("LOG_LEVEL", default)
 
+    def py_log_level(self, default='info'):
+        level = self._get_str("PY_LOG_LEVEL", default)
+        LEVELS = {'trace': logging.NOTSET,
```

Comment:
> What does logging.NOTSET do? 

https://docs.python.org/3.4/library/logging.html#logging.Logger.setLevel

> For sure trace is meaningful for the JS log library we use but I think maybe this should be removed as an option on the python side.

I could be meaningful too from the Python side, but I agree is probably one of the less used levels... I can remove it if you feel is convenient...